include/mbedtls/blowfish.h

```diff
@@ -224,18 +252,24 @@ int mbedtls_blowfish_crypt_cfb64( mbedtls_blowfish_context *ctx,
  *             content must not be written to insecure storage and should be
  *             securely discarded as soon as it's no longer needed.
  *
- * \param ctx           Blowfish context
- * \param length        The length of the data
+ * \param ctx           The Blowfish context to use. This must be initialized
+ *                      and bound to a key.
+ * \param length        The length of the input data in Bytes.
  * \param nc_off        The offset in the current stream_block (for resuming
  *                      within current cipher stream). The offset pointer to
```

Comment:
The line doesn't make sense:
> The offset pointer to should be 0 at the start of a stream and must be smaller than \c 8.

"The offset pointer to should be 0"?

Reply:
Ok.